### PR TITLE
Fixed StackOverflowException when debug in Rider under linux (increas…

### PR DESCRIPTION
**Pull request by oripio**
Fixed StackOverflowException when debug in Rider under linux (increases max stack size)

### CODE REVIEW

File: NBitcoin/Crypto/Cryptsharp/SCrypt.cs [hunk 1]
```diff
@@ -228,7 +228,8 @@ public static class SCrypt
 			Thread[] threads = new Thread[threadCount - 1];
 			for(int i = 0; i < threads.Length; i++)
 			{
-				(threads[i] = new Thread(workerThread, 8192 * 4)).Start();
```
Comment (NicolasDorier) [inline NBitcoin/Crypto/Cryptsharp/SCrypt.cs]: just change `4` to `8`. We are rich

Comment (oripio) [reply NBitcoin/Crypto/Cryptsharp/SCrypt.cs]: Ok but 8192 * 4 != 8192 * 8. 
The PR means the current stack is 32k but when use debugger - 128k.

**Comment by ForNeVeR** [comment]
I should note that the issue is not in Rider, but in .NET Core debugger. For now, one cannot debug that code at all using .NET Core debugger with Rider or VSCode on macOS or Linux.

**Comment by NicolasDorier** [comment]
This code was not compiling, I just fixed it and pushed package 4.1.1.84 let me know

**Comment by oripio** [comment]
Works like a charm. Thanks!

**Pull request closed without merging**